docs/en/modules/admin/pages/participants/verifications_conflicts.adoc

```diff
@@ -1,38 +1,60 @@
-= Verification's conflicts
+= Verifications conflicts
 
-There are cases depending on which kind of verifications you've enabled that there could be verification's conflicts.
+A verification conflict can happen when authorizations are enabled, and some participants were impersonated. 
 
 For instance:
 
-. A participant get xref:admin:participants/impersonations.adoc[impersonated], and they don't provide an email
-. Then they create an account and try to verify themselves with the Census verification
+. A participant get xref:admin:participants/impersonations.adoc[impersonated] with their data, but don't enter an email
+. Then they create an account, and try to verify themselves with the same verification
```

Comment:
This is not the only case a verification conflict can happen. This also happens if:

1. I create an account, and I get verified.
2. Then I create another account and I try to verify myself.

Maybe we should explain this case also, because I think this is the most common case.

Reply:
I updated the documentation to showcase this case, thank youu!